client/js/_author/reducers/preview.js

```diff
@@ -0,0 +1,16 @@
+import * as Assessment from '../../actions/qbank/assessments';
```

Comment:
Turns out this is important. We need to keep the data that comes back from qbank somewhere